merlin/systems/dag/ops/fil.py

```diff
@@ -0,0 +1,91 @@
+import pathlib
+
+import numpy as np
+from google.protobuf import text_format  # noqa
+
+import merlin.systems.triton.model_config_pb2 as model_config  # noqa
```

Comment:
In parallel with your PR, we just switched over to importing this from the tritonclient in #109 to avoid some issues with building/registering classes from Protobuf twice:
```suggestion
import tritonclient.grpc.model_config_pb2 as model_config # noqa
```